src/cargo/core/interning.rs

```diff
@@ -95,3 +97,15 @@ impl PartialOrd for InternedString {
         Some(self.cmp(other))
     }
 }
+
+impl Serialize for InternedString {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        serializer.serialize_str(self.inner)
+    }
+}
+
+unsafe impl Send for InternedString {}
```

Comment:
I think this is a rebase error, it was recently removedin https://github.com/rust-lang/cargo/pull/5287/files#diff-fc6e9b7a46297a8f6d7851cce9978683L94 as a `&'static str` is already send and sync.